oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,428 @@
+===========================
+OEP-55: Project Maintainers
+===========================
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-0055 <oep-0055-proc-project-maintainers>`
+   * - Title
+     - Project Maintainers
+   * - Last Modified
+     - 2022-03-16
+   * - Authors
+     - Edward Zarecor <e0d@tcril.org>, Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Jeremy Bowman <jbowman@2u.com>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2022-01-28
+   * - Review Period
+     - 2022-03-21 - 2022-04-11
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - <links to any other relevant discussions or relevant related materials>
+
+.. contents::
+   :local:
+   :depth: 3
+
+Abstract
+========
+
+The Open edX platform is a service oriented platform for authoring and
+delivering online learning based on the `Django framework`_.  At the
+highest level, the platform is composed of numerous services (IDAs
+[#ida]_), micro-front ends (MFEs [#mfe]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on the GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation,and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered a best practice [#maintainer]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community management elements.  Both dimensions of the role are
+equally important.
+
+All non-deprecated components of the Open edX project must have one
+or more named maintainers.  When no such maintainer(s) exist, this will
+be considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will be open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in any particular community member's production
+environment [#]_.
```

Comment:
I don't see a matching footnote for this footnote. Further, could we please continue using named footnotes? 

```suggestion
environment [#prodenv]_.
```